server/helpers/responseMessage.js

```diff
@@ -1,6 +1,6 @@
-export default (response, statusCode, object) => {
+export default (response, statusCode, object, status = 'failure') => {
```

Comment:
Don't modify this function. You can add the new status to the object you are passing to the function, rather than passing the new status as a separate argument

Reply:
If I add it, it will show the failure status first.

Reply:
Just put it as the first property in the object. It will override the already existing status.

Reply:
Done